examples/criteo-example.ipynb

```diff
@@ -81,15 +115,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
     "# define some information about where to get our data\n",
     "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '/raid/criteo/tests/crit_int_pq')\n",
     "OUTPUT_DATA_DIR = os.environ.get('OUTPUT_DATA_DIR', '/raid/criteo/tests/test_dask') # where we'll save our procesed data to\n",
     "BATCH_SIZE = int(os.environ.get('BATCH_SIZE', 800000))\n",
-    "NUM_PARTS = int(os.environ.get('NUM_PARTS', 2))\n",
```

Comment:
this line needs re-added

Reply:
not sure why the diff isn't showing it but if you look [here](https://github.com/alecgunny/NVTabular/blob/tfasync/examples/criteo-example.ipynb) you'll see I just replaced it with a `PARTS_PER_CHUNK` variable, which is what I use in `tests/integration/test_notebooks.py` as well